docs/tutorials/Setting-up-Raspberry-Pi-Zero-W.md

```diff
@@ -0,0 +1,153 @@
+
+```shell
+sudo apt-get update
+sudo apt-get install -y cmake
+```
+
+### Python 3.4 installation
+Our tutorials require Python 3.4 on the Pi (and Python 3.6 on your computer). Since the last Conda support for ARMV6 is not actively maintained, here we install python 3.4 directly into our system path. We will also install pip3 which is a python package management tool.
+
+To install python 3.4 on your Raspbian Jessie, type the following:
+
+```shell
+sudo apt-get install python3 python3-dev
+wget https://bootstrap.pypa.io/get-pip.py
+sudo python3 get-pip.py
+```
+
+### OpenBLAS
+OpenBLAS is a library for fast linear algebra operations, which can significantly increase the speed of your models. It is optional, but highly recommended. To install OpenBLAS, type the following.
+
+You should not use the libopenblas-dev from the Raspbian distribution channel. Instead, build it from the source to make sure it's built for ARMV6 which is our Raspberry Pi Zero runs on. 
+
+```shell
+git clone https://github.com/xianyi/OpenBLAS.git OpenBLAS
+cd OpenBLAS
+make
+sudo make install
+```
+
+The whole build process will take about 4-5 hours. The above steps will install a working copy of OpenBLAS to /opt/OpenBLAS
+
+### Numpy
+Numpy is a numerical processing package required by OpenCV and our tutorial. Here we will install a copy of Numpy package which uses the OpenBlas we installed in the last step:
+
+```shell
+git clone https://github.com/numpy/numpy.git numpy
+cd numpy
+```
+
+create a new file called sit.cfg under numpy directory and type in following contents:
+
+```shell
+[default]
+include_dirs = /opt/OpenBLAS/include
+library_dirs = /opt/OpenBLAS/lib
+ 
+[openblas]
+openblas_libs = openblas
+library_dirs = /opt/OpenBLAS/lib
+ 
+[lapack]
+lapack_libs = openblas
+library_dirs = /opt/OpenBLAS/lib
+```
+
+now still under the same directory, runt he following commands:
+
+```shell
+python3 setup.py build --fcompiler=gnu95
+python3 setup.py install
+```
+
+### curl
+`curl` is a command line tool used to transfer data via URL. When files are required to be downloaded from a URL, the instructions assume you have `curl` available to perform the download. To install `curl`, type the following:
+```shell
+sudo apt-get install -y curl
+```
+
+Remember to run `python3 xxxx.py` when you run any python scripts from ELL tutorial.
+
+### OpenCV
+[OpenCV](http://opencv.org/) is a computer vision library that enables us to read images from a camera, resize them, and prepare them for processing by ELL. We cannot install it easily through Conda like we do for Raspberry Pi 3 since we don't have a conda tool here, so we have to install it manually again. Now take a deep breath, this will take you lots of time.
+
+Before we start building OpenCV source, we need to install a few tools. Install them by following the steps below:
+
+```shell
+sudo apt-get install build-essential pkg-config
```

Comment:
these apt-get install commands should specify -y

Reply:
Fixed. Thanks!